Gemfile

```diff
@@ -18,6 +18,7 @@ gem "sqlite3"
 gem "turbolinks", "~> 5"
 gem "uglifier", ">= 1.3.0"
 gem "will_paginate", "3.1.6"
+gem 'jquery-rails', '~> 4.3', '>= 4.3.1'
```

Comment:
convention:
- use double quote in Ruby code
- use single quote in Javascript or CSS code